config/routes.rb

```diff
@@ -3,6 +3,6 @@
 
   # Defines the root path route ("/")
   root "home#index"
-  resources :users, only: [:new, :index]
-
+  resources :home, only: [:index]
```

Comment:
Nitpick: Not that it would really matter here, but semantically maybe we want to use `resource` here instead of `resources` since we only have one `Home`. But we're also only using the `index` method so...meh